Docs: Settings updates at runtime (#34169)
grafana/grafana: docs/sources/enterprise/_index.md

- [Vault integration]({{< relref "vault.md" >}}) to manage your configuration or provisioning secrets with Vault.
- [Auditing]({{< relref "auditing.md" >}}) tracks important changes to your Grafana instance to help you manage and mitigate suspicious activity and meet compliance requirements.
- [Request security]({{< relref "request-security.md" >}}) makes it possible to restrict outgoing requests from the Grafana server.

> - [Settings updates at runtime]({{< relref "settings-updates.md" >}}) allows you to update Grafana settings at runtime with no need of restarts.

Reviewer: ```suggestion
- [Settings updates at runtime]({{< relref "settings-updates.md" >}}) allows you to update Grafana settings at runtime without requiring a restart.
```
